bindings/matrix-sdk-crypto-ffi/src/lib.rs

```diff
@@ -471,14 +473,6 @@ mod uniffi_types {
     pub use crate::{backup_recovery_key::BackupRecoveryKey, machine::OlmMachine};
 }
 
-#[allow(warnings)]
-mod generated {
```

Comment:
Why was this renamed / moved into its own file?

Reply:
This is to use the same uniffi setup as `matrix-rust-sdk` as well as [official guide](https://mozilla.github.io/uniffi-rs/tutorial/Rust_scaffolding.html) when we do not seem to be using custom location.